test/lib/clients/vault_test.rb

```diff
@@ -18,15 +20,15 @@ class VaultTest < ActiveSupport::TestCase
     vault_client.sys.unmount(intermediate_ca_mount)
   end
 
-  test "#configure_kv" do
-    @client.stub :kv_mount, intermediate_ca_mount do
+  test ".configure_kv" do
+    @client.stub :kv_mount, kv_mount do
       assert @client.configure_kv
       engines = vault_client.sys.mounts
-      assert_equal "kv", engines[intermediate_ca_mount.to_sym].type
+      assert_equal "kv", engines[kv_mount.to_sym].type
     end
   end
 
-  test "#configure_pki" do
+  test ".configure_pki" do
```

Comment:
what is the diff between '#' and '.'?

is there convention here that I should understand?

Reply:
It's a convention from `rspec` which I adopted for the descriptions here -- https://grosser.it/2010/06/18/how-to-name-rspec-describe-blocks-for-methods/